playbooks/roles/plugins/foreman_docker/tasks/main.yml

```diff
@@ -0,0 +1,69 @@
+- name: 'Prepare hammer taxonomy options'
+  set_fact:
+    foreman_docker_hammer_taxonomy_params: "--organizations '{{ foreman_docker_organization }}' --locations '{{ foreman_docker_location }}'"
+  when: foreman_docker_use_taxonomies
+
+- name: 'Prepare blank hammer taxonomy options'
+  set_fact:
+    foreman_docker_hammer_taxonomy_params: ""
+  when: not foreman_docker_use_taxonomies
+
+- name: "Install foreman docker plugin"
+  yum:
```

Comment:
I'm not sure about the reasoning, but to introduce it, adding one line to [this file](https://github.com/Katello/katello-installer/blob/master/config/katello-answers.yaml#L74) and a migration for previous installations would enabled it.